deployment-docs/README.md

```diff
@@ -61,3 +61,56 @@ For example:
 See the [taskcluster-web-server](../services/web-server#readme) documentation for details on the avaialble login strategies.
 
 The `ui_login_strategy_names` variable should be equal to the set of keys in `ui_login_strategies`, but *must not contain secrets* as it is sent to the browser.
+
+## GitHub app setup
+
+This may be obsoleted when bstack lands manifest stuff. First spin up your Taskcluster instance in order to get the URLs required at app creation and an arbitrary secret. Then follow these steps to set the app up. 
+
+### Why are there 2 apps? 
+
+One of them is what Taskcluster uses to get notified of changes on github ("the Taskcluster GitHub app"), and update github statuses ("the Taskcluster login app"). The other pertains to GitHub login and is only needed on Taskcluster instances that use this feature.
+
+If you use a GitHub app to sign in, the token it recieves can do anything as the user who just signed in. That's why Taskcluster keeps this feature as isolated from everything else as possible. 
```

Comment:
@owlishDeveloper does this accurately reflect what you determined a few months ago?

Reply:
yes